lib/matplotlib/table.py

```diff
@@ -558,6 +590,10 @@ def table(ax,
           rowLabels=None, rowColours=None, rowLoc='left',
           colLabels=None, colColours=None, colLoc='center',
           loc='bottom', bbox=None, edges='closed',
+          cellEdgeColour=None,
```

Comment:
I'm British, I wrote the original code, way back in the day.   Times were different then.